doc/http-administration-api-documentation.md

```diff
@@ -0,0 +1,5 @@
+# MongooseIM's HTTP Administration API
+
+Find the beautiful Swagger documentation below:
+
+<iframe src="https://cdn.rawgit.com/esl/MongooseIM/swagger-http-api-doc/doc/swagger/index.html?url=https://raw.githubusercontent.com/esl/MongooseIM/swagger-http-api-doc/priv/http_api/backend_swagger.yaml#/" align="middle" height="700" width="700"></iframe>
```

Comment:
Serving the Swagger UI of off GitHub Pages is another option instead the proxy at `https://cdn.rawgit.com/esl/MongooseIM/swagger-http-api-doc/doc/swagger/index.html`.